devito/function.py

```diff
@@ -1432,8 +1476,8 @@ def _interpolation_indices(self, variables, offset=0):
             mapper = {}
             for j, d in zip(idx, self.grid.dimensions):
                 p = points[j]
-                lb = sympy.And(p > d.symbolic_start - self._radius, evaluate=False)
-                ub = sympy.And(p < d.symbolic_end + self._radius, evaluate=False)
+                lb = sympy.And(p >= d.symbolic_start - self._radius, evaluate=False)
+                ub = sympy.And(p <= d.symbolic_end + self._radius, evaluate=False)
```

Comment:
why?

Reply:
because otherwise I don't access values in the last halo entry, which are instead available